docs/syncer.md

```diff
@@ -86,4 +86,41 @@ kcp-syncer   1/1     1            1           13m
 
 1. Wait for the kcp sync target to go ready.
 
-TODO(marun)
+```sh
+$ kubectl get synctarget -o jsonpath='{.items[0].status.conditions}' | jq .
+[
+  {
+    "lastTransitionTime": "2022-08-15T14:54:35Z",
+    "status": "True",
+    "type": "Ready"
+  },
+  {
+    "lastTransitionTime": "2022-08-15T14:54:35Z",
+    "status": "True",
+    "type": "HeartbeatHealthy"
+  }
+]
+```
+
+1. Create a deployment
+
+```sh
+$ kubectl create deployment kuard --image gcr.io/kuar-demo/kuard-amd64:blue
+```
+
+Note: replace "gcr.io/kuar-demo/kuard-amd64:blue" with "gcr.io/kuar-demo/kuard-arm64:blue" in case you're running
+an Apple M1 based virtual machine.
+
+Verify the deployment on the local workspace:
+
+```sh
+$ kubectl get deployment
+NAME    READY   UP-TO-DATE   AVAILABLE   AGE
+kuard   1/1     1            1           100s
+```
+
+Verify the deployment on the physical kind cluster:
+```sh
+$ kubectl --kubeconfig $KIND_KUBECONFIG get deployment -A | grep kuard
```

Comment:
You should not *need* to do this, right?

Reply:
eh, that's true, let's leave it out